doc/source/api/solver/fielddata.rst

```diff
@@ -226,6 +226,22 @@ The response to a pathlines field request contains the following fields:
 - ``particle-time``, which contains  particle time, if requested. 
 - ``additional field name``, which contains  additional field, if requested. additional field name is the same name as the additional field name passed in the request.
 
+Allowed values
+--------------
+Additionally there is an allowed_values provided on field name,
+surface name and ids to guide the users.
+
+Some sample use cases are demonstrated below:
+
+.. code-block:: python
+
+  >>> field_data.get_scalar_field_data.field_name.allowed_values()
+  >>> transaction.add_scalar_fields_request.field_name.allowed_values()
+  >>> field_data.get_scalar_field_data.surface_name.allowed_values()
+  >>> transaction.add_scalar_fields_request.surface_names.allowed_values()
```

Comment:
What is a transaction?  It's not defined by this example so I cannot just copy/paste it to my console and run it.

Reply:
Thank you @dnwillia-work, this has been updated.